x-pack/plugins/lens/public/indexpattern_datasource/format_column.ts

```diff
@@ -117,13 +117,27 @@ export const formatColumn: ExpressionFunctionDefinition<
           }
           if (parentFormatParams) {
             const innerParams = (col.meta.params?.params as Record<string, unknown>) ?? {};
-            return withParams(col, {
-              ...col.meta.params,
-              params: {
-                ...innerParams,
-                ...parentFormatParams,
-              },
-            });
+            if (isNestedFormat(col.meta.params)) {
```

Comment:
What do you think of `isNestedFormat(innerParams)` here?
I find it easier to read/parse, but not particular strong opinion here.